app/models/concerns/hyrax/file_set_behavior.rb

```diff
@@ -24,13 +24,15 @@ module FileSetBehavior
       self.human_readable_type = 'File'
 
       attribute :file_identifiers, Valkyrie::Types::Set
+      attribute :file_nodes, Valkyrie::Types::Set.member(FileNode.optional)
```

Comment:
What does this mean?  Is FileNode embeded in the FileSet?  Is that going to work in Fedora?